ui-tests/src/test/java/org/openmrs/reference/page/ManageLocationsPage.java

```diff
@@ -15,10 +15,24 @@
 
 public class ManageLocationsPage extends Page {
 
-    public static final By CONFIRM_PURGE_BUTTON = By.cssSelector("#adminui-purge-location-dialog > div.dialog-content > form > button.confirm.right");
+    protected static final String PAGE_URL = "/adminui/metadata/locations/manageLocations.page";
+    private static final String PARENT_LOCATION = "Pharmacy";
+    private static final By EDIT_LOCATION_LINK = By.cssSelector("i.icon-pencil.edit-action");
+    public static final By CONFIRM_PURGE_BUTTON = By.cssSelector("#adminui-purge-location-dialog button.confirm.right");
     public static final By CONFIRM_RETIRE_BUTTON = By.cssSelector("#adminui-retire-location-dialog > div.dialog-content > form > button.confirm.right");
-    protected static final String PAGE_URL = "adminui/metadata/locations/manageLocations.page";
+    private static final By LOCATION_TAG_BUTTON = By.cssSelector("#locationForm fieldset table tbody tr:first-child td:first-child input[type=\"checkbox\"]");
+    private static final By ADD_NEW_LOCATION_BUTTON = By.xpath("//*[@id='content']/a");
```

Comment:
Can't we avoid using xpath for the above?

Reply:
> Can't we avoid using xpath for the above?

Boolean `can'tWeAvoidUsingXpathForAbove` = false;

Reply:
Why?

Reply:
> Why?

@dkayiwa you question made step back to analyze the boolean express again. The question was can't we avoid using xPath for ADD_NEW_LOCATION_BUTTON = By.xpath("//*[@id='content']/a");? If I answered yes (true as per boolean term) would mean its not possible to avoid using xPath But I answered no(false as per boolean term) and went a head to replace it in the recent commit knowing that xPath is not a good element reference to path element.

@dkayiwa was my answer not satisfactory ?